aiocouchdb/tests/test_feeds.py

```diff
@@ -108,19 +108,16 @@ def test_buffer_workflow(self):
         self.assertTrue(feed.is_active())
 
         result = yield from feed.next()
-        self.assertEqual(feed._queue.qsize(), buf_size)
```

Comment:
Probably, because it causes test failure. Whole this test is sort of flaky with new Python/asyncio versions.